opencti-platform/opencti-front/tests_e2e/dashboard/dashboard.spec.ts

```diff
@@ -4,9 +4,17 @@ import DashboardPage from '../model/dashboard.pageModel';
 import DashboardDetailsPage from '../model/dashboardDetails.pageModel';
 import DashboardFormPage from '../model/form/dashboardForm.pageModel';
 import DashboardWidgetsPageModel from '../model/DashboardWidgets.pageModel';
+import TopMenuProfilePage from '../model/menu/topMenuProfile.pageModel';
+import LoginFormPageModel from '../model/form/loginForm.pageModel';
 import LeftBarPage from '../model/menu/leftBar.pageModel';
+import AccessRestrictionPageModel from '../model/AccessRestriction.pageModel';
 import MalwareDetailsPage from '../model/malwareDetails.pageModel';
 
+// Because of login/logout stuff in access restriction test below, running
+// both in parallel make conflicts.
+// Need to be resolved in an other way but for now we do it like this.
```

Comment:
I didn't know this tag feature, could be useful indeed. Maybe it can help us run some in parallel but in that case I think tags should be more like @serie or @parallel more than the name of the feature?

For the init part, even without tag it runs before any test